openapi.yml

```diff
@@ -55,16 +93,81 @@ components:
       properties:
         id:
           type: string
+        category:
+          description: High level categorization of the material, such as Book, Article, Journal.
+          type: string
+        source_link:
+          type: string
+          format: uri
+        realtime_holdings_link:
+          type: string
+          format: uri
+        sfx_link:
+          type: string
+          format: uri
+        year:
+          type: string
         title:
           type: string
+        full_text_links:
+          type: array
+          items:
+            $ref: '#/components/schemas/Link'
+        authors:
```

Comment:
Added ticket to discuss and decide on author/creator/contributor models with metadata folks [https://mitlibraries.atlassian.net/browse/DIP-120](https://mitlibraries.atlassian.net/browse/DIP-120)